README.md

```diff
@@ -210,19 +210,23 @@ printed to STDOUT.
 Valid debug levels are: `DEBUG`, `INFO`, `WARN`, `DEBUG`,
 `CRITICAL`, `ERROR`.
 
+The `--info` switch prints the total number of process executions and the total
+number of in-process iterations. It also prints the benchmark keys which will
+be skipped.
+
 ## Running in reboot and resume modes
```

Comment:
Having read up to here, it isn't quite clear to me whether `--info` should be used with a config file or a results file (and Krun now takes either as an argument)